core/templates/pages/blog-dashboard-page/blog-dashboard-page.module.ts

```diff
@@ -0,0 +1,72 @@
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Module for the blog-dashboard page.
+ */
+
+import { HttpClientModule, HTTP_INTERCEPTORS } from '@angular/common/http';
+import { APP_INITIALIZER, DoBootstrap, NgModule } from '@angular/core';
+import { BrowserModule } from '@angular/platform-browser';
+import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
+import { downgradeComponent, downgradeModule } from '@angular/upgrade/static';
+import { SharedComponentsModule } from 'components/shared-component.module';
+
+import { OppiaAngularRootComponent } from 'components/oppia-angular-root.component';
+import { BlogDashboardNavbarBreadcrumbComponent } from 'pages/blog-dashboard-page/navbar/blog-dashboard-navbar-breadcrumb.component';
+import { platformFeatureInitFactory, PlatformFeatureService } from 'services/platform-feature.service';
+import { RequestInterceptor } from 'services/request-interceptor.service';
+
+declare var angular: ng.IAngularStatic;
+
+@NgModule({
+  imports: [
+    BrowserModule,
+    HttpClientModule,
+    SharedComponentsModule
+  ],
+  declarations: [
+    OppiaAngularRootComponent,
+    BlogDashboardNavbarBreadcrumbComponent,
+  ],
+  entryComponents: [
+    OppiaAngularRootComponent,
+    BlogDashboardNavbarBreadcrumbComponent
+  ],
+  providers: [
+    {
+      provide: HTTP_INTERCEPTORS,
+      useClass: RequestInterceptor,
+      multi: true,
+    },
+    {
+      provide: APP_INITIALIZER,
+      useFactory: platformFeatureInitFactory,
+      deps: [PlatformFeatureService],
+      multi: true,
+    },
+  ],
+})
+class BlogDashboardPageModule implements DoBootstrap {
+  ngDoBootstrap() {}
+}
+
+angular.module('oppia').requires.push(downgradeModule(extraProviders => {
+  const platformRef = platformBrowserDynamic(extraProviders);
+  return platformRef.bootstrapModule(BlogDashboardPageModule);
+}));
+
+angular.module('oppia').directive('oppiaAngularRoot', downgradeComponent({
+  component: OppiaAngularRootComponent,
+}));
```

Comment:
You haven't used AngularJS anywhere, right? You can actually remove AngularJS from these files. You can refer to the other pages in the `templates/pages` directory or you can refer to #13073 or #13098.

Reply:
I will be using schema based editor in the dashboard, thus currently we cannot remove AngularJs from the module as the editor is yet not migrated.

Reply:
Schema based editors are not migrated yet. If they are going to be used on this page, angularjs is required.

Reply:
@ashutoshc8101 Thanks for the confirmation.